### PR TITLE
Fix link pointing to https://nlu.rasa.com

### PR DESCRIPTION
**Pull request by federicotdn**
**Proposed changes**:
- Fix an old link pointing to https://nlu.rasa.com

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

**Comment by federicotdn** [comment]
There another one [here](https://github.com/RasaHQ/rasa/blob/master/rasa/core/nlg/callback.py#L126) but I'm not sure what to change it to.

**Comment by erohmensing** [comment]
@federicotdn looks like the second one should go to a currently non-existent api documentation for the NLG server: https://github.com/RasaHQ/rasa/issues/2961 (so i guess right now we could just link to the spec page)

as for the first one, should we merge into 1.2.x?

**Comment by federicotdn** [comment]
Ok! Should we first merge it into master though?

**Comment by erohmensing** [comment]
Yes good point that's fine 👍

**Comment by federicotdn** [comment]
@erohmensing Please review this PR when you get the chance. Thanks!

**Comment by erohmensing** [comment]
https://github.com/RasaHQ/rasa/blob/master/rasa/core/nlg/callback.py#L12

File: rasa/nlu/extractors/crf_entity_extractor.py [hunk 1]
```diff
@@ -158,7 +158,7 @@ def _check_spacy_doc(self, message):
                 "message {}\n"
                 "POS features require a pipeline component "
                 "that provides `spacy_doc` attributes, i.e. `SpacyNLP`. "
-                "See https://nlu.rasa.com/pipeline.html#nlp-spacy "
+                "See https://rasa.com/docs/rasa/nlu/choosing-a-pipeline/#pretrained-embeddings-spacy "
```
Comment (erohmensing) [inline rasa/nlu/extractors/crf_entity_extractor.py]: We should probably use the DOCS_BASE_URL env var for this

Comment (federicotdn) [reply rasa/nlu/extractors/crf_entity_extractor.py]: Good idea, didn't know about that variable

**Pull request merged**